jobs/resource_job.go

```diff
@@ -823,6 +823,21 @@ func ResourceJob() *schema.Resource {
 			if js.isMultiTask() {
 				ctx = context.WithValue(ctx, common.Api, common.API_2_1)
 			}
+
```

Comment:
So that this Update function does not get humongous, would you mind pulling this into a separate function like `prepareJobSettingsForUpdate`?

Reply:
Done.